packages/dds/tree/src/test/rebaserAxiomaticTests.ts

```diff
@@ -380,6 +654,196 @@ function verifyComposeAssociativity<TChangeset>(
 	assertDeepEqual(rightPartialCompositions.at(-1), singlyComposed);
 }
 
+function verifyComposeWithInverseEqualsEmpty<TChangeset>(
+	edits: TaggedChange<TChangeset>[],
```

Comment:
It seems like this function should take a single edit instead of an array.

Reply:
Updated to take in a single edit